sandbox/middleware-play.example

```diff
@@ -0,0 +1,184 @@
+<!DOCTYPE html>
+<html lang="en-GB">
+<head>
+  <meta charset="utf-8" />
+  <title>Video.js Sandbox</title>
+
+  <!-- Add ES5 shim and sham for IE8 -->
+  <script src="../build/temp/ie8/videojs-ie8.js"></script>
+
+  <!-- Load the source files -->
+  <link href="../build/temp/video-js.css" rel="stylesheet" type="text/css">
+  <script src="../dist/video.js"></script>
+  <script src="../node_modules/videojs-flash/dist/videojs-flash.js"></script>
+
+  <!-- Set the location of the flash SWF -->
+  <script>
+    videojs.options.flash.swf = '../build/temp/video-js.swf';
+  </script>
+
+  <style>
+    .terminate-btn {
+      margin: 2em 1em;
+    }
+
+    .vjs-progress-control.vjs-control .vjs-play-progress.vjs-slider-bar.terminated {
```

Comment:
why not look for the `terminated` class on the player instead?

Reply:
```css
.terminated .vjs-process-control .vjs-play-progress
```
Then below (line 44), you can just do `player.addClass('terminated')` instead of going through the entire child heirarchy to get the proper element. Or something like that.